doc/man1/openssl-verification-options.pod

```diff
@@ -574,87 +579,128 @@ keyCertSign bit set if the keyUsage extension is present.
 
 =head3 Extended Key Usage
 
-The extKeyUsage (EKU) extension places additional restrictions on the
-certificate uses. If this extension is present (whether critical or not)
-the key can only be used for the purposes specified.
-
-A complete description of each check is given below. The comments about
+The extKeyUsage (EKU) extension places additional restrictions on
+certificate use. If this extension is present (whether critical or not)
+in an end-entity certficiate, the key is allowed only for the uses specified,
+while the special EKU B<anyExtendedKeyUsage> allows for all uses.
+
+Note that according to RFC 5280 section 4.2.1.12,
+the Extended Key Usage extension will appear only in end-entity certificates,
+and consequently the standard certification path validation described
+in its section 6 does not include EKU checks for CA certificates.
+The CA/Browser Forum requires for TLS server, S/MIME, and code signing use
+the presence of respective EKUs in subordinate CA certificates (while excluding
+them for root CA certificates), which is self-contradictory because OTOH they
+take over the certificate validity concept and path validation from RFC 5280.
```

Comment:
Please remove: `, which is self-contradictory because OTOH they
take over the certificate validity concept and path validation from RFC 5280.`
I do not thing it belongs here.

Reply:
I believe it is important to keep that the CA/Browser Forum does not define their own cert validity and validation (but just place requirements on cert contents).
So changed to:
"while taking over from RFC 5280 the certificate validity concept and certificate path validation."